libslide/src/bignum/utils.rs

```diff
@@ -64,6 +97,27 @@ pub fn ifft(item: Vec<Complex>, n: usize) -> Vec<Complex> {
     a
 }
 
+// converts a multi digit polynomial to an single digit vector
```

Comment:
Also
```suggestion
/// converts a multi digit polynomial to an single digit vector
```
this creates a rustdoc comment that will be picked up by our docs creator :)